app/services/RedirectService.scala

```diff
@@ -56,6 +64,118 @@ object RedirectService {
       )
   }
 
+  def carBenefitsRedirects(cya: EmploymentCYAModel, taxYear: Int, employmentId: String): Seq[ConditionalRedirect] = {
```

Comment:
im not sure if i get the use of this method, is it not just calling the one above?

Reply:
yeah I added it so the carBenefits page could call this one, but you're right could simply call above method. Should I remove this one?